DQM/L1TMonitor/python/L1TStage2EMTFEmul_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+l1tStage2EmtfEmul = cms.EDAnalyzer(
+    "L1TStage2EMTF",
+    emtfSource = cms.InputTag("valEmtfStage2Digis", "EMTF"),
```

Comment:
Use different InputTags for the different collections you are going to use in the module to avoid ProductNotFound exceptions